test-requirements.txt

```diff
@@ -9,3 +9,6 @@ ordereddict; python_version <= '2.6'
 logutils; python_version <= '2.6'
 unittest2; python_version <= '2.6'
 Twisted==12.2; python_version <= '2.6'
+# flake8 3 dropped support for py{26,33}
```

Comment:
I'm fine only running flake8 on 2.7 since it's for developers and we should all have at least 2.7